src/test/java/nextstep/subway/acceptance/LineSectionAcceptanceTest.java

```diff
@@ -73,6 +135,67 @@ void removeLineSection() {
         assertThat(response.jsonPath().getList("stations.id", Long.class)).containsExactly(강남역, 양재역);
     }
 
+    /**
+     * Given 지하철 노선에 새로운 구간 추가를 요청 하고
+     * When 지하철 노선의 중간 구간 제거를 요청 하면
+     * Then 노선에 구간이 제거된다
+     */
+    @DisplayName("지하철 노선에 구간을 제거")
+    @Test
+    void removeLineSection_When_중간_역을_제거하면_Then_중간_구간제거() {
+        // given
+        Long 정자역 = 지하철역_생성_요청("정자역").jsonPath().getLong("id");
+        지하철_노선에_지하철_구간_생성_요청(신분당선, createSectionCreateParams(양재역, 정자역, 10));
+
+        // when
+        지하철_노선에_지하철_구간_제거_요청(신분당선, 양재역);
+
+        // then
+        ExtractableResponse<Response> response = 지하철_노선_조회_요청(신분당선);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertThat(response.jsonPath().getList("stations.id", Long.class)).containsExactly(강남역, 정자역);
+    }
+
+    /**
+     * Given 지하철 노선에 구간이 하나뿐일때
+     * When 구간 제거를 요청 하면
+     * Then 구간이 제거되지 않는다
+     */
+    @DisplayName("지하철 노선에 구간을 제거")
+    @Test
+    void removeLineSection_Given_노선에_구간이_하나뿐일때_When_구간을_제거하면_Then_BAD_REQUEST() {
+        // when
+        ExtractableResponse<Response> response = 지하철_노선에_지하철_구간_제거_요청(신분당선, 양재역);
```

Comment:
```suggestion
        ErrorResponse response = 지하철_노선에_지하철_구간_제거_요청(신분당선, 양재역);
```

ErrorResponse를 넘겨주면 더 간결해지지 않을까요?